server/app.js

```diff
@@ -23,9 +24,26 @@ app.use(cookieParser());
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(express.static(path.join(__dirname, 'public')));
 
-const MongoClient = require('mongodb').MongoClient;
-const uri = `mongodb://${process.env.DBUSER}:${process.env.DBPASSWORD}@${process.env.DBHOST}/${process.env.DB}`;
-const client = new MongoClient(uri);
+/* 
+ * If testDB is true, uses a locally hosted mongoDB
+ * Otherwise, it will use a cloud hosted DB set in the .env file
+ * MongoDB must be installed
+ */
+const testDB = true;
+
+/* Mongoose Setup */
+const mongoose = require('mongoose');
+const uri = testDB ? 'mongodb://localhost:27017/ofi-testdata' : `mongodb://${process.env.DBUSER}:${process.env.DBPASSWORD}@${process.env.DBHOST}/${process.env.DB}`;
+mongoose.connect(uri, {useNewUrlParser: true});
```

Comment:
Do we need an `await` here? (after we move it into `main())

Reply:
Yes- it's been added